use_case_examples/cifar_brevitas_training/README.md

```diff
@@ -93,11 +93,10 @@ Experiments were conducted on an m6i.metal machine offering 128 CPU cores and 51
 | ---------------------- | -------- | -------- |
 | VGG Torch              | None     | 88.7     |
 | VGG FHE (simulation\*) | None     | 88.7     |
-| VGG FHE (simulation\*) | 8 bits   | 88.3     |
-| VGG FHE (simulation\*) | 7 bits   | 88.3     |
-| VGG FHE (simulation\*) | 6 bits   | 87.5     |
-| VGG FHE (simulation\*) | 5 bits   | 84.9     |
-| VGG FHE                | 6 bits   | 87.5\*\* |
+| VGG FHE (simulation\*) | 8 bits   | 88.0     |
+| VGG FHE (simulation\*) | 7 bits   | 87.2     |
+| VGG FHE (simulation\*) | 6 bits   | 86.0     |
+| VGG FHE                | 6 bits   | 86.0\*\* |
```

Comment:
I haven't checked the 10 samples in FHE yet

Reply:
all is good ! -> https://github.com/zama-ai/concrete-ml/actions/runs/6121822384/job/16616458716